### PR TITLE
Some design tweaks

### PR DESCRIPTION
**Pull request by faiza-a**

### CODE REVIEW

File: docs/_includes/header.html [hunk 1]
```diff
@@ -2,6 +2,9 @@
 <nav class="navbar" >
     <div class="container">
         <div class="navbar-brand">
+            {% for logo in page.header %}
+            <img src="{{ logo.logo_image }}" height="90" width="90">
```
Comment (jtomeck) [inline docs/_includes/header.html]: I think all you need is the width attribute. If you'd prefer to have both attributes, I would set height to "auto"

Comment (faiza-a) [reply docs/_includes/header.html]: I actually tried both. didn't work for me. could you try this on your side too?

Comment (faiza-a) [reply docs/_includes/header.html]: worked now. thanks

Comment (faiza-a) [reply docs/_includes/header.html]: done

**Pull request merged**